src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -733,4 +733,7 @@ Do you want to proceed?</value>
   <data name="CursorMustBeOnAMemberName" xml:space="preserve">
     <value>Cursor must be on a member name.</value>
   </data>
+  <data name="FontAndColors_BraceMatching" xml:space="preserve">
+    <value>Brace Matching</value>
```

Comment:
We'll now have "Brace Matching", "Brace Matching (Highlight)" and "Brace Matching (Rectangle)" next to each other in Fonts & Colors. Any opinions on how better to distinguish are very welcome. All my ideas are problematic:

- "Brace Matching (C#/VB)" - bad because TypeScript and the list may grow
- "Brace Matching (Roslyn)" - can't refer to it as Roslyn, though it's accurate
- "Brace Matching (Combined)" - leaves the user wondering why the uncombined ones exist still